controller.py

```diff
@@ -1,75 +1,187 @@
 #!/usr/bin/env python
+import sys
+import os
+path = os.path
+sys.path.append('uavDy/')
+sys.path.append('animation/')
```

Comment:
avoid that. Instead it should be possible to use relative imports? (`from animation import ...`)

Reply:
Yes it is possible to use relative imports, I will do it. 
I also added in the readme to add the crazyflie-firmware to the PYTHONPATH (as in the main branch readme) and I added a "try: ... except" for the import of cffirmware in the "if __name__ == '__main__':" condition. I think it is better to be added manually, because if someone is using another firmware code in another path.